cliquet/views/hello.py

```diff
@@ -20,6 +21,9 @@ def get_hello(request):
     if eos:
         data['eos'] = eos
 
+    if git and git.revision:
+        data['commit'] = git.revision
```

Comment:
``TypeError: fatal: Not a git repository (or any parent up to mount point /home)``